app/models/petition.rb

```diff
@@ -293,6 +293,17 @@ def mark_petitions_as_debated!(date = Date.current)
 
     private
 
+    def popular_in(constituency_id, count)
+      klass = ConstituencyPetitionJournal
+      constituency_signature_count = klass.arel_table[:signature_count].as('constituency_signature_count')
+      constituency_signatures_for = klass.with_signatures_for(constituency_id).ordered
+
+      select(arel_table[Arel.star], constituency_signature_count).
+      joins(:constituency_petition_journals).
+      merge(constituency_signatures_for).
+      limit(count)
+    end
```

Comment:
This is definitely better than the string interpolation version we had before. :clap: